src/test/java/kitchenpos/application/MenuGroupServiceTest.java

```diff
@@ -0,0 +1,73 @@
+package kitchenpos.application;
+
+import kitchenpos.dao.MenuGroupDao;
+import kitchenpos.domain.MenuGroup;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(MockitoExtension.class)
+@DisplayName("MenuGroupService 클래스 테스트")
+public class MenuGroupServiceTest {
+    @Mock
+    private MenuGroupDao menuGroupDao;
+    @InjectMocks
+    private MenuGroupService menuGroupService;
+
+    @Nested
+    @DisplayName("create 메서드 테스트")
+    public class CreateMethod {
+        @Test
+        @DisplayName("메뉴그룹 생성 성공")
+        public void success() {
+            // given
+            final MenuGroup mockMenuGroup = setupSuccess("test menu group");
+
+            // when
+            final MenuGroup createdMenuGroup = menuGroupService.create(mockMenuGroup);
+
+            // then
+            assertThat(createdMenuGroup.getName()).isEqualTo(mockMenuGroup.getName());
+        }
+
+        private MenuGroup setupSuccess(String name) {
+            final MenuGroup mockMenuGroup = new MenuGroup();
+            mockMenuGroup.setName("test menu group");
+            Mockito.when(menuGroupDao.save(Mockito.any())).thenReturn(mockMenuGroup);
```

Comment:
```suggestion
            BDDMockito.given(menuGroupDao.save(Mockito.any())).willReturn(mockMenuGroup);
```
BDDMockito를 사용하면 given when then 스타일에 맞게 작성할 수 있어요 😀